docs/pages/blog/mui-core-v5.md

```diff
@@ -0,0 +1,560 @@
+  props: ButtonUnstyledProps,
+  ref: React.ForwardedRef<any>,
+) {
+  const { children } = props;
+  const { active, disabled, focusVisible, getRootProps } = useButton({
+    ...props,
+    ref,
+    component: CustomButtonRoot,
+  });
+
+  const classes = {
+    active,
+    disabled,
+    focusVisible,
+  };
+
+  return (
+    <CustomButtonRoot {...getRootProps()} className={clsx(classes)}>
+      {children}
+    </CustomButtonRoot>
+  );
+});
+```
+
+<p class="blog-description"><a href="https://codesandbox.io/s/7lc1r?file=/demo.tsx">Codesandbox</a></p>
+
+We discuss the effort in [#6218](https://github.com/mui-org/material-ui/issues/6218).
+You can use [#27170](https://github.com/mui-org/material-ui/issues/27170) to follow our progress.
+
+## Improved DX
+
+### Smaller demos in the docs
+
+We have used the migration of the demos from JSS to emotion as an opportunity to rework them.
+Many of the demos were added taking into account how they would help maintainers work on the components.
+We have reversed the priority, putting the developers using them [first](https://github.com/mui-org/material-ui/issues/22484).
+
+In practice, it means that we have focused on breaking down complex demos into smaller ones.
+We aim for having as many "inline preview" as possible. It saves one click to expand the demo and the mental overhead of figuring out the part of the code that map with what you saw interesting on the screen.
+
+<a href="/components/buttons/#basic-button"><img src="/static/blog/mui-core-v5/inline-preview.png" alt="" style="width: 649px; margin-bottom: 16px;" /></a>
+
+### Props descriptions in IntelliSense
+
+The best documentation is the one you don't need to open.
+We have moved all the props' descriptions to TypeScript, so the IntelliSense in your editor can show you more context.
+
+<img src="/static/blog/mui-core-v5/prop-descriptions.png" alt="" style="width: 649px; margin-bottom: 16px;" />
+
+<p class="blog-description">The popup explains what the <code>forcePopupIcon</code> prop is for.</p>
+
+These TypeScript props' descriptions are also used to generate the [API pages](/api/autocomplete/#props) of the documentation, so there is a single source of truth.
+
+### Migration from Enzyme to Testing Library
+
+- The value? Makes it easier for developers to copy the source, and maintainers to make sure the components are easy to test
+
+### TypeScript migration
+
+- Share our progress https://github.com/mui-org/material-ui/issues/15984
+- Covered a bit in https://material-ui.com/blog/2020-q3-update/
+- The value? Helps keeping the types as accurate as possible
+
+### Strict Mode support
+
+- Docs, test, etc. now runs in StrictMode
+- Warn that `@material-ui/styles` is not compatible
+
+## A new product line: X
+
+- What are the new product separation
+- What this product line is about
+- Why does it even exist
+- Present the new Data Grid component, hopefully, released as v5.0.0 at the same time.
```

Comment:
What is the deadline for the release of the v5 of the core ?
For the Grid, I suppose we won't release v5 before mid-October as we only launch the first beta at mid-September and we have a week of partial-break.

Reply:
> What is the deadline for the release of the v5 of the core ?

Tomorrow, the blog post is about MUI Core now, this comment is about outdated.